templates/geosafe/analysis/modal/impact_card.html

```diff
@@ -18,9 +18,9 @@ <h4 class="modal-title" id="impact_card_label">{% trans 'Analysis Result' %}</h4
                     {% if summary.analysis_question %}
                         {{ summary.analysis_question }}
                     {% elif summary.exposure_type != 'polygon people' and summary.exposure_type != 'population' %}
-                        In the event of {{ analysis.hazard_layer.title }}, the number of {{ summary.exposure_type }}s in hazard areas in {{ analysis.exposure_layer.title }} that are likely to be affected is
+                        {% trans 'In the event of {{ analysis.hazard_layer.title }}, the number of {{ summary.exposure_type }}s in hazard areas in {{ analysis.exposure_layer.title }} that are likely to be affected is' %}
```

Comment:
CMIIW, if you want to use `{{  }}` expressions inside, you had to use block trans https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#blocktrans-template-tag